### ISSUE TITLE
Weather Script not Working

### ISSUE THREAD

**Issue by navanchauhan** [alexanderepstein/Bash-Snippets#149]
**Issue Label:**
* [x] Bug
* [ ] New feature
* [ ] Enhancement
* [ ] New component

**Description:**
Shows no output 

### If its a bug make sure to include this section.
**OS and OS version:**
* [ ] Mac
* [ ] Linux 32 Bit
* [x] Linux 64 Bit 
* [ ] Windows 32 Bit
* [ ] Windows 64 Bit

OS Version:
Kali Linux 2017

Screenshot :
![Screenshot](https://i.imgur.com/7jO0UCQ.png)

**Comment by retokromer** [comment]
FYI: On my Ubuntu 16.04 LTS is works fine.

**Comment by navanchauhan** [comment]
@retokromer @alexanderepstein  I tried it yesterday on Parrot Sec OS, it didn't work. Today I installed Kali and still it doesn't work

**Comment by navanchauhan** [comment]
output with set -x
```
+ currentVersion=1.20.0
+ LANG=C.UTF-8
++ echo C.UTF-8
++ cut -c1-2
+ locale=C.
+ configuredClient=
+ getConfiguredClient
+ command -v curl
+ configuredClient=curl
+ getopts uvh opt
+ [[ 0 == \0 ]]
+ checkInternet
+ httpGet github.com
+ getIPWeather
++ httpGet ipinfo.io/country
++ case "$configuredClient" in
++ curl -A curl -s ipinfo.io/country
+ country=IN
+ [[ IN == \U\S ]]
++ httpGet ipinfo.io/loc
++ case "$configuredClient" in
++ curl -A curl -s ipinfo.io/loc
+ location=28.6833,77.0667
+ httpGet C..wttr.in/28.6833,77.0667
+ case "$configuredClient" in
+ curl -A curl -s C..wttr.in/28.6833,77.0667
+ exit 1

```

**Comment by navanchauhan** [comment]
@alexanderepstein My OSes did not have values for locale
Recommend replacing $locale with $country (Got it working after doing this)

**Comment by alexanderepstein** [comment]
Huh I am not sure why your locale is just being set to C. when I  ```echo $LANG``` I get ```en_US.UTF-8``` the idea is to get your countries version of the weather in terms of language ex. ```curl fr.wttr.in``` outputs the weather in french. I guess I can check if the locale has not been set to something valid and just set it to ```en``` but this is more of a workaround rather than a fix.

**Comment by alexanderepstein** [comment]
Fixed with commit 47e91be373e2099f22e2e9a06c3fb0603f70414a
